datasette_dashboards/__init__.py

```diff
@@ -92,7 +92,7 @@ async def dashboard_view(request, datasette):
             database = datasette.get_database(db)
         except KeyError:
             raise NotFound(f"Database does not exist: {db}")
-        await check_permission_execute_sql(request, datasette, database)
+        await check_permission_execute_sql(request, datasette, db)
```

Comment:
This trigger a linting issue with Flake8:

```
datasette_dashboards/__init__.py:92:13: F841 local variable 'database' is assigned to but never used
```

Could you fix it?

Reply:
Oh, sure.

Reply:
Sorry but the whole `try...except` block is now useless

Reply:
you're probably right, the permission check will fail if the DB does not exist anyway

Reply:
Actually removing the `try` will make `test_dashboard_view_unknown_chart_db` fail, because now it doesn't return a 404 anymore for a non-existent DB, but a 200.

I need to come back to this tomorrow to do it properly then.

Reply:
No rush, thanks for trying to resolve the issue. I too need to take some time to explore and understand those API changes in the upcoming release of Datasette 1.0.0